app/models/form/sales/questions/buyer2_working_situation.rb

```diff
@@ -26,6 +26,15 @@ def answer_options
         "6" => { "value" => "Not seeking work" },
         "7" => { "value" => "Full-time student" },
         "8" => { "value" => "Unable to work due to long term sick or disability" },
+        "9" => {
+          "value" => "Child under 16",
+          "depends_on" => [
+            { "saledate" => { "operator" => "<", "operand" => Time.zone.local(2024, 4, 1) } },
```

Comment:
If `form.start_year_2025_or_later?` the saledate can't be before 2024?

Maybe it's worth having answer_options and displayed_answer_options as we do on some other questions?